lib/litani.py

```diff
@@ -239,6 +239,13 @@ def add_jobs_to_cache():
     jobs = []
     cache_dir = get_cache_dir()
     jobs_dir = cache_dir / JOBS_DIR
+    if not os.path.exists(jobs_dir):
```

Comment:
`jobs_dir` is a `pathlib.Path`, I suggest
```
if not jobs_dir.exists()
```

Also, please don't raise below. Controlled terminations should be done through `sys.exit(1)` in Litani.

I think the error message is a bit verbose... for stuff like this, I think having a very constrained error message describing (1) what the error is and (2) how to avoid it, in that order, is helpful. The error message below makes it sound like the user should create a directory... I don't think the directory (or what its name is) is relevant, it's actually an implementation detail that users aren't even aware about.

I suggest something like "Cannot run build: no jobs were added. Run 'litani add-job' one or more times before running 'litani run-build'".

Reply:
Good points on the verbosity and controlled termination. Seeing a `FileNotFoundError` exception being raised is unnecessarily more dirty. Now:

```bash
➜ python3 litani init --project-name nyx --stages aether
Report will be rendered at file:///var/folders/bd/w7pff05n5k35735rjvph7dkh0000gs/T/litani/runs/latest/html/index.html
➜ python3 litani run-build
litani: Cannot run build: no jobs were added. Run `litani add-job` one or more times before running `litani run-build`.
```